python/src/robyn/modeling/feature_engineering.py

```diff
@@ -19,6 +19,15 @@
 from robyn.data.entities.hyperparameters import Hyperparameters, ChannelHyperparameters
 from robyn.data.entities.mmmdata import MMMData
 
+# Set logging level for this module
+logging.getLogger("cmdstanpy").setLevel(logging.CRITICAL)  # Only show critical errors
```

Comment:
What does this and next line do? logger for this class is instantiated later, which will not affect with these two lines